### ISSUE TITLE
Keyboard focus is visible on two places simultaneously under 'page' popup.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#101]
User Experience: 

If focus is visible on two places simultaneously under ‘Pages’ popup, then the user with mobility impairment who rely on keyboard  can become disoriented and will face difficulty to interact with UI element.

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode.

4. Navigate to the Left nav and select Pages button.

5. Navigate to the various controls of the "Pages" window.

6. Verify if default focus is visible on two places simultaneously or not.

 

Actual Result:

Keyboard focus is visible on two places at same time on list items under ‘Pages’ popup.

 

Expected Result:

Keyboard focus should not be visible on two places at same time under ‘Pages’ popup.

Note:

Similar issue exist with Chrome.

Similar issue exist throughout the application.

 

MAS Reference:

MAS 2.4.7 - Focus Visible (05A)

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by azaslonov** [comment]
Fixed.
